jmetal-core/src/main/java/org/uma/jmetal/util/experiment/component/GenerateWilcoxonTestTablesWithR.java

```diff
@@ -156,14 +156,14 @@ private void printLines(GenericIndicator<?> indicator, String rFileName, String
           "    if (is.finite(wilcox.test(data1, data2)$p.value) & wilcox.test(data1, data2)$p.value <= 0.05) {" + "\n" +
           "      if (median(data1) <= median(data2)) {" + "\n" +
           "        write(\"$\\\\blacktriangle$\", \"" + latexFileName + APPEND_STRING +
-          "      }" + "\n" +
```

Comment:
I would not only remove the `+`, but also all the `"\n"` by using a [`StringBuilder`](https://docs.oracle.com/javase/8/docs/api/java/lang/StringBuilder.html). It would make it also a lot more readable and the auto-formatting would be more resilient.